pkg/ccl/sqlccl/restore.go

```diff
@@ -541,7 +545,14 @@ func presplitRanges(baseCtx context.Context, db client.DB, input []roachpb.Key)
 		// EnsureSafeSplitKey for more context.
 		splitKey := append([]byte(nil), splitPoints[splitIdx]...)
 		splitKey = keys.MakeRowSentinelKey(splitKey)
-		if err := db.AdminSplit(ctx, splitKey); err != nil {
+		const maxAttempts = 3
+		if err := retry.WithMaxAttempts(ctx, base.DefaultRetryOptions(), maxAttempts, func() error {
+			err := db.AdminSplit(ctx, splitKey)
+			if err != nil {
+				log.Warningf(ctx, "failed to split range at %s: %s", splitKey, err)
+			}
+			return err
+		}); err != nil {
```

Comment:
(This change is unrelated; fixes an ambiguous result error that's been present for a while.)